### ISSUE TITLE
publishing with scripts - topic empty 

### ISSUE THREAD

**Issue by gatovelho** [kamilfb/mqtt-spy#71]
Hello,
I  am using mqtt-spy-050 on WIndows 10 32 bits home .

After opening (mqtt and firing) a connection window on my local broker, the publish message topic
is empty (methinks: starting with my default publish topic would be useful, but this is not the issue).

Then, after  selecting  a publishing script and try to use it by "publishing with script",
I get a popup saying "cannot publish to an empty topic". On inspection, I find the topic
is embedded into the script itself, so it should not be needed to insert any topic into the
publish window. In fact it is not even used for publishing, because we can insert any.

We can live with this, it is not a real problem. 

Best Regards,
CS

**Comment by kamilfb** [comment]
That should be fixed in the latest beta - could you see if that's any better now?

**Comment by gatovelho** [comment]
```
Hi Kamil,

i can't work now.

But I will check it soon.

Best regards

Às 20:07 de 13-05-2016, Kamil
  Baczkowicz escreveu:


  That should be fixed in the latest beta - could you see if
    that's any better now?
  —
    You are receiving this because you authored the thread.
    Reply to this email directly or view
      it on GitHub









-- 
```

I’m open to any suggestions, I’ve been wrong before.

**Comment by kamilfb** [comment]
This is now available in 0.5.1. Let me know if you find any further issues!
